ios/FluentUI/Tooltip/TooltipView.swift

```diff
@@ -81,10 +81,15 @@ class TooltipView: UIView {
         updateAppearance(tokenSet: tokenSet)
     }
 
+    override func traitCollectionDidChange(_ previousTraitCollection: UITraitCollection?) {
+        layoutSubviews()
```

Comment:
only should do the layout if there is a change that is relavant? can you test as you change ipad's compact to regular mode, rotation, size change? video capture on the description?